lib/tmuxinator.rb

```diff
@@ -13,9 +13,9 @@
 require "tmuxinator/project"
 require "tmuxinator/window"
 require "tmuxinator/version"
+require "tmuxinator/tmux_validator"
 
-module Tmuxinator
-end
+module Tmuxinator; end
```

Comment:
Good point. I'll back that out and it can be formatted/removed elsewhere, if people feel strongly about it.